bundles/org.openhab.binding.gce/src/main/java/org/openhab/binding/gce/internal/handler/Ipx800v3Handler.java

```diff
@@ -0,0 +1,273 @@
+                                    }
+                                }
+                            } else {
+                                state = OpenClosedType.OPEN;
+                                if (!portData.isInitializing()) {
+                                    triggerPushButtonChannel(channel, EVENT_RELEASED);
+                                    if (config2.longPressTime != 0 && sinceLastChange < config2.longPressTime) {
+                                        triggerPushButtonChannel(channel, EVENT_SHORT_PRESS);
+                                    }
+                                }
+                            }
+                            break;
+                        case RELAY_OUTPUT:
+                            state = value == 1 ? OnOffType.ON : OnOffType.OFF;
+                            break;
+                    }
+                }
+                updateState(channel.getUID().getId(), state);
+                if (!portData.isInitializing()) {
+                    updateState(channel.getUID().getId() + PROPERTY_SEPARATOR + LAST_STATE_DURATION_CHANNEL_NAME,
+                            new QuantityType<>(sinceLastChange / 1000, SmartHomeUnits.SECOND));
+                }
+                portData.setData(value, now);
+            } else {
+                logger.debug("Received data '{}' for not configured port '{}'", value, port);
+            }
+        } else {
+            logger.debug("Received data '{}' for not configured channel '{}'", value, port);
+        }
+    }
+
+    protected void triggerPushButtonChannel(Channel channel, String event) {
+        logger.debug("Triggering event '{}' on channel '{}'", event, channel.getUID());
+        triggerChannel(channel.getUID().getId() + PROPERTY_SEPARATOR + CHANNEL_TYPE_PUSH_BUTTON_TRIGGER, event);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Received channel: {}, command: {}", channelUID, command);
+
+        String channelName = channelUID.getIdWithoutGroup();
+        Channel channel = thing.getChannel(channelUID.getId());
+        if (channelName.chars().allMatch(Character::isDigit) && RELAY_OUTPUT.equalsIgnoreCase(channelUID.getGroupId())
+                && command instanceof OnOffType && channel != null) {
+            RelayOutputConfiguration config = channel.getConfiguration().as(RelayOutputConfiguration.class);
+            parser.ifPresent(p -> p.setOutput(channelName, (OnOffType) command == OnOffType.ON ? 1 : 0, config.pulse));
+            return;
+        }
+        logger.info("Can not handle command '{}' on channel '{}'", command, channelUID);
+    }
+
+    @Override
+    public void channelLinked(ChannelUID channelUID) {
+        logger.debug("channelLinked: {}", channelUID);
+        final String channelId = channelUID.getId();
+        if (channelUID.getIdWithoutGroup().chars().allMatch(Character::isDigit)) {
```

Comment:
Maybe put this check in a method with the channelUID as parameter. The check is done in multiple places. It looks like you could also simply check if it contains `-` or not or 
```suggestion
        if (!channelUID.getIdWithoutGroup().contains("-")) {
```